language/de/acp/extensions.php

```diff
@@ -71,6 +71,12 @@
 	'EXTENSION_NAME'			=> 'Name der Erweiterung',
 	'EXTENSION_ACTIONS'			=> 'Vorgänge',
 	'EXTENSION_OPTIONS'			=> 'Optionen',
+	'EXTENSION_INSTALL_HEADLINE'=> 'Eine Erweiterung installieren',
+	'EXTENSION_INSTALL_EXPLAIN'	=> '<ol>
+			<li>Lade die Erweiterung aus der phpBB Erweiterungsdatenbank herunter</li>
+			<li>Entpacke die Erweiterung und lade sie in das <samp>ext/</samp> Verzeichnis deines phpBB Boards hoch.</li>
+			<li>Aktiviere die Erweiterung hier in der Erweiterungs-Verwaltung</li>
```

Comment:
Das ist der erste "hier"-Bezug in phpBB (auch im Englischen) :-?
Ich würde das hier einfach rausschmeißen. Wir sind bei solchen Anleitungen immer sehr kurz geblieben (siehe EXTENSION_UPDATE_EXPLAIN)